### PR TITLE
BZ2025396: Added known issue to 4.8 release notes

### PR DESCRIPTION
**Pull request by ctauchen**
Added a known issue to the 4.8 release notes.

https://bugzilla.redhat.com/show_bug.cgi?id=2025396

Preview (see final entry under Known issues): https://deploy-preview-39209--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes.html#ocp-4-8-known-issues

### CODE REVIEW

**Comment by sinnykumari** [comment]
LGTM

**Comment by reestr** [comment]
LGTM

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -2480,6 +2480,8 @@ $ oc annotate -n <NAMESPACE> route/<ROUTE-NAME> "haproxy.router.openshift.io/bal
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=2017708[*BZ#2017708*])
 
+* During upgrades from an {product-name} 4.8.z release to a later 4.8.z release, if the images for :op-system: and the Machine Config Operator do not change, the upgrade is marked as complete before the control plane nodes have finished upgrading. Users who perform operations on their cluster before the upgrade is actually complete may introduce errors and cause the upgrade to fail. As a workaround, verify that updates are completed on the control plane nodes, using the command `oc get mcp/master`, before performing additional operations on the cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2025396[*BZ#2025396*])
```
[the remark this answers is not in the document]
Comment (pneedle-rh) [reply release_notes/ocp-4-8-release-notes.adoc:2483]: You could direct the second sentence to the reader. For example:

"Following this, the upgrade might fail if you perform operations on your cluster before the upgrade is actually complete."

[the remark this answers is not in the document]
Comment (pneedle-rh) [reply release_notes/ocp-4-8-release-notes.adoc:2483]: You could split the final sentence into two, to facilitate localisation and to enable an explicit description of what the command does. For example:

"As a workaround, verify that updates are completed on the control plane nodes before performing additional operations on the cluster. You can use the  `oc get mcp/master` command to review the status of the MCO-managed nodes that are available on your cluster for each pool."

The use of "MCO" in the example assumes that you have described the acronym previously in this section.

**Review by pneedle-rh — COMMENTED** [review]
@ctauchen I have now completed my peer review. I have added a few suggestions. Other than those, this LGTM!

File: release_notes/ocp-4-8-release-notes.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (ctauchen) [reply release_notes/ocp-4-8-release-notes.adoc:2483]: Exellent suggestions all. I've taken every one. Thanks!

**Pull request merged**